src/infra/log/zap/logger.go

```diff
@@ -87,21 +87,7 @@ func (l Logger) WithComponent(component string) log.Logger {
 }
 
 func (l *Logger) Write(p []byte) (n int, err error) {
-	switch l.cfg.Level {
```

Comment:
Did you validate this change? IIRC this implementation was intended to only Write logs if level corresponds to the expected one. Otherwise you will write logs even if the defined logger level does not match

Reply:
yes I validated this, it was a bug before that printed the access logs at the same level as the config.